src/Penrose/Shapes.hs

```diff
@@ -322,30 +322,30 @@ type ComputedValue a = ([Property], [Value a] -> Value a)
 computedProperties ::
      (Autofloat a) => M.Map (ShapeTypeStr, Property) (ComputedValue a)
 computedProperties =
```

Comment:
This was noted in #166 before.

Reply:
Again, my guess is that the polymorphism forces the dict to be constructed repeatedly

Reply:
We should really look into this... maybe time to create a minimal example? I still don't get why the function & objective (etc) dictionaries don't cause a huge polymorphism slowdown (unless they do and it doesn't show up in the profile)

Reply:
Yeah, feel free to note this in one of the performance-related issues (e.g. #166 #213) and we can pick a time to look into it.